OnboardPietroPieri/package.json

```
@@ -12,9 +12,16 @@
   "dependencies": {
     "@apollo/client": "^3.8.8",
     "@react-native-async-storage/async-storage": "^1.21.0",
+    "@react-navigation/stack": "^6.3.20",
```

Poderia fixar a versão
```suggestion
    "@react-native-async-storage/async-storage": "1.21.0",
    "@react-navigation/stack": "6.3.20",
```